src/compat.h

```diff
@@ -8,6 +8,7 @@
 #define TIMESCALEDB_COMPAT_H
 
 #include <postgres.h>
+#include <pgstat.h>
```

Comment:
why this change?

Reply:
Yeah, so `WaitLatchCompat` requires `PG_WAIT_EXTENSION`, which requires the includes.
Because this is a macro, the compiler doesn't complain in `src/compat.h`, but rather in any file in which the macro gets preprocessed. I just thought it's easier to add this includes inside `compat.h`, than in all the files that might use `WaitLatchCompat`, but I'm open to arguments against.

Reply:
👍